site/src/self_profile.rs

```diff
@@ -37,9 +40,15 @@ pub fn generate(
         ProcessorType::Crox => {
             let opt = serde_json::from_str(&serde_json::to_string(&params).unwrap())
                 .context("crox opts")?;
+            let data = crox::generate(self_profile_data, opt).context("crox")?;
+
+            let mut encoder = GzEncoder::new(Vec::new(), Compression::default());
```

Comment:
Could you please experiment with the `Compression` levels (there is also a constructor `Compression::fast()` and `Compression::best()`) to see what is the performance/result size trade-off? Maybe we could find some better options than the defaults.